fifa_draft/models.py

```diff
@@ -135,13 +143,13 @@ def validate_image(self):
         Player, blank=True, related_name="pending_player"
     )
 
-    def __str__(self):
+    def __str__(self) -> str:
         return self.name
 
     class Meta:
         ordering = ["created"]
 
-    def image_url(self):
+    def image_url(self) -> ImageFieldFile:
```

Comment:
Pytanie, czy na pewno `ImageFieldFile`? W tym pierwszym przypadku może – nie zagłębiałem się, bo nie mam u siebie repo i nie debugowałem. Ale po excpecie masz po prostu stringa. Czyli, jeżeli w 1 przypadku faktycznie zwraca `ImageFieldFIle`, a w drugim na pewno stringa, to powinno być `Union[ImageFieldFile, str]` (`from typing import Union`, a od pythona 3.9 wystarczyłoby `ImageFieldFile | str`).

Inna kwestia, że chyba lepiej byłoby zmienić tak tę funkcję, żeby w obu przypadkach zwracała ten sam typ.

Reply:
Z tego co debugowałem tak jak pokazywałeś to taki typ zwraca, uzupełnię o str i sprawdzę czy można tak zrobić żeby ten sam typ był w obu przypadkach.